usability_webhooks/controllers/main.py

```diff
@@ -37,25 +38,38 @@ def _create_api_logs(self, model, vals, function):
             # Error from odoo exception, rollback all data (if config in system parameter)
             if rollback_except:
                 request.env.cr.rollback()
-        request.env["api.log"].create(data_dict)
+        if vals["is_create_logs"]:
+            request.env["api.log"].create(data_dict)
         return res
 
+    def _set_is_create_logs(self, param, vals):
```

Comment:
```suggestion
    def _set_create_logs(self, param, vals):
```

พยายามตั้งชื่อให้สั้น ตัวไหนไม่จำเป็นไม่ต้องใช้ก็ได้